app/assets/stylesheets/application.scss

```diff
@@ -13,3 +13,5 @@
  *= require_tree .
  *= require_self
  */
+
+@import 'bootstrap/scss/bootstrap'
```

Comment:
When installing and starting the application in a completely new host, this line produces an error. Please fix this. I suspect it must be some missing gem that is causing this mistake. To reproduce this error you may clone your own repo in a completely new folder, run `bundle install` there, then `rails db:create` `rails db:migrate` and `rails s`, and then try to open your homepage at `localhost:3000` to see the error message.

Reply:
Thank you @voscarmv for the constructive feedback.  Regarding the error produced when starting the application in a completely new host, one can run "yarn add bootstrap jquery popper.js" and this will solve the problem.